### DISCUSSION TITLE
empty vertical space creates new line

### DISCUSSION THREAD

**Discussion by paul2t** [typst/typst#4645] [General]
### Description

Adding an empty vertical space creates a gap between two lines.
A bigger gap than between two normal lines.

```
line 1\
#v(0pt)
line 2
```
![image](https://github.com/user-attachments/assets/d303f8e6-558a-4902-bfd0-ee72ebff2a2b)

```
line 1
#v(0pt)
line 2
```
![image](https://github.com/user-attachments/assets/d303f8e6-558a-4902-bfd0-ee72ebff2a2b)

```
line 1\
line 2
```
![image](https://github.com/user-attachments/assets/42b13357-e633-4a28-a164-11ddebc40806)

```
line 1
line 2
```
![image](https://github.com/user-attachments/assets/23f05c4a-f5f5-4fa1-8153-c8fdcc98f030)

### Operating system

Web app

### Typst version

- [X] I am using the latest version of Typst

**Comment by Leedehai** [comment]
Hi - a non-weak `v(0em)` is intended to be like a `parbreak()` based on the examples of https://typst.app/docs/reference/layout/v/.

One could disagree about this design and I think it also makes sense.. but that would make it a feature request imo.

**Comment by paul2t** [comment]
Hi, thanks for the response.
Indeed, I didn't notice it was like a `parbreak()`
Intuitively, it makes sense to me that an empty vertical space doesn't move the cursor. But I understand that some people could disagree. There might be a legitimate use case where the `parbreak` behavior by default makes more sense.
Also, changing the behavior could break some existing documents.
I don't know how to change this to a feature request though.
Should I create a new issue ? Or is it possible to change the label ?
